include/rocksdb/advanced_cache.h

```diff
@@ -13,7 +13,9 @@
 #include <string>
 
 #include "rocksdb/cache.h"
+#include "rocksdb/compression_type.h"
 #include "rocksdb/memory_allocator.h"
+#include "rocksdb/options.h"
```

Comment:
Is this needed?

Reply:
I had to include it for the definition of `CacheTier`.